src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.subway.favorite.application;
+
+import nextstep.subway.auth.domain.UserDetail;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class FavoriteService {
+    private FavoriteRepository favoriteRepository;
+    private StationRepository stationRepository;
+
+    public FavoriteService(FavoriteRepository favoriteRepository, StationRepository stationRepository) {
+        this.favoriteRepository = favoriteRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    public FavoriteResponse saveFavorites(UserDetail userDetail, FavoriteRequest request) {
```

Comment:
LoginMember로 받는건 어떨까요?

```suggestion
    public FavoriteResponse saveFavorites(LoginMember loginMember, FavoriteRequest request) {
```